### PR TITLE
Test fix Database fix. Server fix

### PR DESCRIPTION
**Pull request by BluePowerPlayer**

### CODE REVIEW

File: AcceptanceTests/TestUni.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ public async Task Create_Uni_AndGetUni()
             string newGuid = Guid.NewGuid().ToString();
             var uni = new University()
             {
-                Name = "Vilniaus Universitetas",
+                Name = Guid.NewGuid().ToString(),
```
Comment (Doombringer-MarAlith) [inline AcceptanceTests/TestUni.cs]: Name = guid ?

File: App/Program.cs [hunk 2]
```diff
@@ -18,8 +18,13 @@ internal static class Program
         /// </summary>
         [STAThread]
         private static void Main()
-        { 
-            
+        {
+
```
Comment (Doombringer-MarAlith) [inline App/Program.cs]: Unnecessary line

File: App/Program.cs [hunk 3]
```diff
@@ -18,8 +18,13 @@ internal static class Program
...
+            var a = Container.GetService<FormManagerData>();
+            var aa = Container.GetService<FormManagerData>();
+            a.CurrentReviewIndex = 5;
+            var g = a.CurrentReviewIndex;
+            var ga = aa.CurrentReviewIndex;
```
Comment (Doombringer-MarAlith) [inline App/Program.cs]: What is this

File: Dbo/AddJsons.cs [hunk 4]
```diff
@@ -12,7 +12,7 @@ public class AddJsonFiles
     {
         private static readonly string SqlConnectionString = DatabaseExecutor.ConnectionString;
 
-        private static readonly DatabaseExecutor db = new DatabaseExecutor();
+        private static readonly DatabaseExecutor Db = new DatabaseExecutor();
```
Comment (Doombringer-MarAlith) [inline Dbo/AddJsons.cs]: Tai pervadinam i `Database`. Full names ftw

File: AcceptanceTests/TestUni.cs (hunk 1 above)
Comment (BluePowerPlayer) [reply AcceptanceTests/TestUni.cs]: it is a test. to ensure that this is unique we use guids in tests

File: App/Program.cs (hunk 3 above)
Comment (BluePowerPlayer) [reply App/Program.cs]: WOOOPS

**Pull request merged**